### DISCUSSION TITLE
mysql_config not found

### DISCUSSION THREAD

**Discussion by ginnyedelstein** [apache/airflow#14108] [Q&A]
Hi,

I'm getting "mysql_config not found" when using the mysql extra dependency (with python 3.6 and poetry)

**Comment by XD-DENG** [comment]
@ginnyedelstein may you please more detailed information? Like

- which version of Airflow you are using? What OS?
- what were you executing when you get this error? Installing Airflow, or starting webserver/scheduler, or during running a DAG?
- more importantly, may you provide the detailed traceback?

**Reply by ginnyedelstein** [reply]
using 1.10.14, and i get it when trying to install the dependencies from the toml file. I was getting the same error upon `db init` when trying to use mysqlclient, but was able to use mysql-connector-python and pymysql successfully. Do you know what the mysql extra installs?

**Reply by XD-DENG** [reply]
Please refer to https://github.com/apache/airflow/blob/master/setup.py#L360
